samples/dnn/inpainting.cpp

```diff
@@ -0,0 +1,203 @@
+/*
+This sample inpaints the masked area in the given image.
+
+Copyright (C) 2025, Bigvision LLC.
+
+How to use:
+    Sample command to run:
+
+        ./example_dnn_inpainting
+    The system will ask you to draw the mask on area to be inpainted
+
+    You can download lama inpainting model using:
+       `python download_models.py lama`
```

Comment:
I propose to add reference to the original model and how it was converted, if it's not onnx or simplification was applied.